src/commands/split_column.rs

```diff
@@ -1,26 +1,40 @@
+use crate::commands::WholeStreamCommand;
 use crate::errors::ShellError;
 use crate::object::{Primitive, TaggedDictBuilder, Value};
 use crate::prelude::*;
 use log::trace;
 
-pub fn split_column(
-    args: CommandArgs,
-    registry: &CommandRegistry,
-) -> Result<OutputStream, ShellError> {
+pub struct SplitColumn;
+
+impl WholeStreamCommand for SplitColumn {
+    fn run(
+        &self,
+        args: CommandArgs,
+        registry: &CommandRegistry,
+    ) -> Result<OutputStream, ShellError> {
+        split_column(args, registry)
+    }
+
+    fn name(&self) -> &str {
+        "split-column"
+    }
+
+    fn signature(&self) -> Signature {
+        // TODO: Signature?
+        // TODO: Improve error. Old error had extra info:
+        //
+        //   needs parameter (e.g. split-column ",")
+        Signature::build("split-column").required("delimeter", SyntaxType::Any)
```

Comment:
I believe this should also have a `.rest()` at the end after the required since `split-column` can take the column names after the delimiter.